addons/advanced_throwing/functions/fnc_drawArc.sqf

```diff
@@ -50,7 +50,7 @@ for "_i" from 0.05 to 1.45 step 0.1 do {
             _cross = 1; // 1: Distance Limit (Green)
         } else {
             // Even vanilla throwables go through glass, only "GEOM" LOD will stop it but that will also stop it when there is glass in a window
-            if (lineIntersects [_prevTrajASL, _newTrajASL]) then { // Checks the "VIEW" LOD
+            if (count (lineIntersectsSurfaces [_prevTrajASL, _newTrajASL]) > 0) then { // Checks the "VIEW" LOD
```

Comment:
Use `isEqualTo`.

Advanced throwing doesn't work underwater, only drop mode, due to water resistance. So glass underwater is irrelevant.

Reply:
I think `lineIntersectsSurfaces` checks 2 different LODs, so it might not be the same as the original command

Reply:
^ It checks FIRE and VIEW, I confirmed that it worked in the exact same way, and who says you don't want to drop a grenade through a window facing up while underwater?

Reply:
There, should be good to go, other uses of lineIntersects or lineIntersectsWith mostly don't need to be replaced.